packages/micropip/src/micropip/_micropip.py

```diff
@@ -412,42 +422,53 @@ async def install(
         A requirement or list of requirements to install. Each requirement is a
         string, which should be either a package name or URL to a wheel:
 
-        - If the requirement ends in ``.whl`` it will be interpreted as a URL.
-          The file must be a wheel named in compliance with the
-          `PEP 427 naming convention <https://www.python.org/dev/peps/pep-0427/#file-format>`_.
+        - If the requirement ends in ``.whl``, the part of the requirement after
+          the last ``/``  must be a valid wheel name in compliance with the `PEP
+          427 naming convention.
+
+        - If the requirement ends in ``.whl`` and starts with ``emfs:``, it will
+          be interpreted as a path in the Emscripten file system (Pyodide's file
+          system). E.g., `emfs:../relative/path/wheel.whl` or
+          `emfs:/absolute/path/wheel.whl`.
+
+        - If the requirement ends in ``.whl`` and does not start with ``emfs:``
+          it will be interpreted as a URL. The file must be a wheel named in
+          compliance with the `PEP 427 naming convention
+          <https://www.python.org/dev/peps/pep-0427/#file-format>`_.
 
-        - If the requirement does not end in ``.whl``, it will interpreted as the
-          name of a package. A package by this name must either be present in the
-          Pyodide repository at :any:`indexURL <globalThis.loadPyodide>` or on PyPI
+        - If the requirement does not end in ``.whl``, it will interpreted as
+          the name of a package. A package with this name must either be present
+          in the Pyodide lock file or on PyPI.
```

Comment:
```suggestion
        - Otherwise, it will be interpreted as
          the name of a package. A package with this name must either be present
          in the Pyodide lock file or on PyPI.
```